src/gettingstarted/template/create-project.md

```diff
@@ -0,0 +1,49 @@
+# Start with a template
+
+## Create a new project from a template
+
+In the previous step, you set up a free trial account with Platform.sh. Now you have access to the Platform.sh [management console](/administration/web.md).
+
+From here you can create projects, adjust account settings, and a lot more that you will explore throughout these Getting Started guides.
+
+<video controls autoplay loop>
+  <source src="/videos/management-console/create-project-mc.mp4" type="video/mp4">
+</video>
```

Comment:
Don't auto-play it.  Moving video distracts the viewer from the text on the page.  I doubt we want any videos to autoplay in this guide.

Reply:
I'm not sure if they should loop, either.  Open to discussion on this one.

Reply:
No to autoplays on vids and asciis